### DISCUSSION TITLE
Cleared Symbol colour, dedicated button to show cleared, Category for Scheduled

### DISCUSSION THREAD

**Discussion by Mreco99** [actualbudget/releases#251] [Feature ideas]
Finally found this hidden place. 
There are so many 'places' for Actualbudget, as a normal NEW user, its a mine field (Slack, Trello, Github, Reddit, Spectrum). Github wasn't even on the support page. Please update it for others. 

1. An actual dedicated button to "Only show uncleared" Or a toggle, instead of using the filter, every....single....time. 
a little button in the header of the column would be sooo welcome. Plus when I reconcile, cleared entries dont hide, I have to FILTER again to hide cleared. unintuitive. 

2. Change the colour of the circled ticks (for the cleared and uncleared) they look too similar when scrolling down the list. Green for cleared (as is) and red or white for uncleared.

3. Im wasting time adding categories to scheduled entries every time they get current. Surely just adding a category to a scheduled transaction once is simple? (ok nothing is simple I know)

**Comment by Kidglove57** [comment]
I never hide reconciled transactions, in YNAB or anywhere else, so I will pass on that one!

As regards your (3). It's a bit long winded but here is how to automatically categorise. Go back to the scheduled transaction, click on the button "edit as a rule". You can now add extra actions including a category or a note. Tip - always save a scheduled transaction before going back in to "edit as a rule".

**Comment by Deguello** [comment]
Adding a reconciled option with the ability, but not the requirement, to hide them has been suggested. Hopefully in the works.
